guides/micronaut-oauth2-auth0/micronaut-oauth2-auth0.adoc

```diff
@@ -0,0 +1,123 @@
+include::{commondir}/common-header.adoc[]
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+== OAuth 2.0
+
+To provide authentication, sign in to your https://auth0.com[Auth0] account.
+
+Create an Application
+
+image::auth0-create-application.png[]
+
+Fill the Application URIs:
+
+image::auth0-application-uris.png[]
+
+- Enter `http://localhost:8080/oauth/callback/auth0` as callback url.
+
+- Enter `http://localhost:8080/logout` as allowed logout url.
+
+- Enter `http://localhost:8080` as allowed web origins.
+
+Once you have an https domain (e.g. `https://myapp.org`) enter it as Application Login URI: `https://myapp.org/oauth/login/auth0`.
+
+You can obtain the application's domain, client id and secret in the Auth0 console. 
+
+image::auth0-clientid-clientsecret.png[]
+
+We want to use an **Authorization Code** grant type flow which it is described in the following diagram:
+
+image::diagramm.png[]
+
+
+include::{commondir}/common-create-app.adoc[]
+
+=== Dependencies
+
+To use OAuth 2.0 integration, add the following dependency:
+
+dependency:micronaut-security-oauth2[groupId=io.micronaut.security]
+
+Also add https://micronaut-projects.github.io/micronaut-security/latest/guide/index.html#jwt[Micronaut JWT support] dependencies:
```

Comment:
Use a caret symbol to make the link open in a new window, and remove unnecessary index.html

```suggestion
Also add https://micronaut-projects.github.io/micronaut-security/latest/guide/#jwt[Micronaut JWT support^] dependencies:
```

Reply:
I don't we we should open links in new windows. I think we should let the reader choose and use the default. Our audience is tech savy. If they want to open links in new windows or tabs they know how to do that. I don't think we should use `target=”_blank”` in our links.